src/hipchatbot.js

```diff
@@ -0,0 +1,88 @@
+import Hipchatter from 'hipchatter';
+import trebek from './trebek';
+import winston from 'winston';
+import App from './models/App';
+import { HIPCHAT_CALLBACK_URL } from './config';
+
+export default class HipchatBot {
+  constructor(express) {
+    this.express = express;
+    this.start();
+  }
+
+  async start() {
+    this.app = await App.get();
+    this.hipchatter = new Hipchatter(this.app.apiToken);
+    this.registerWebhook();
+    this.express.post('/webhook', this.onMessage.bind(this));
+  }
+
+  onMessage(req) {
+    const { 
+      event: subtype,
+      item: { 
+        room: { name: channel_name, }, 
+        message: { 
+          message: text, 
+          from: { 
+            id: user_id, 
+            mention_name: user_name,
+          },
+        },
+      },
+    } = req.body;
+
+    const channel_id = (req.body.item.room.id).toString();
+    const timestamp = Date.parse(req.body.item.message.date) / 1000;
```

Comment:
The left hand assignment above didn't like these expressions. Anyway, looks cleaner than before, thanks!